app/services/feeds/optimized.rb

```diff
@@ -0,0 +1,279 @@
+module Feeds
+  class Optimized
```

Comment:
"Optimized" feels a little vague here. Is this optimized for performance? Is it optimized for user experience? user preference? etc. 

Maybe something along these lines that just gives a little more color as to what we are optimizing for.
`HighVolumeOptimized` `Articles::VolumeOptimized`

I would also vote to keep `Articles` in the namespace. I think that makes it very clear what is in the Feed. In the future I could see having a user feed, event feed, comment feed etc and then we would have to redo this.

Reply:
100% agree with @mstruve on the naming of this service here